RELEASE.md

```diff
@@ -51,7 +51,8 @@ on https://appstoreconnect.apple.com :
 
 8. for a **Regular release**, open "My Apps/Delta Chat iOS/iOS App+ (first item)"
    a) enter the version number (without leading "v")
-   b) fill out "what's new", use CHANGELOG.md as a template
+   b) fill out "what's new", use CHANGELOG.md as a template, add the line:
+      "These features will roll out over the coming days. Thanks for using Delta Chat!"
```

Comment:
We could add something like "If you want to get them now, please head to the App Store and download the latest (and greatest) Delta Chat manually". This is a way for users to bypass the phased release

Reply:
hm, was also expecting it to work like that, however, last time we tried that, we did not find a  reliable way to force an update. but maybe it has changed since then - or i messed sth. up.